files/brews/hub.rb

```diff
@@ -0,0 +1,26 @@
+require 'formula'
+
+class Hub < Formula
+  homepage "http://hub.github.com/"
+
+  if RUBY_PLATFORM =~ /x86_64/
+    url "https://github.com/github/hub/releases/download/v2.2.0-rc1/hub_mac_amd64_v2.2.0-rc1.tar.gz"
+    sha1 "413ce80d32f6313cef010e0edb1e664e1fff7427"
+  else
+    url "https://github.com/github/hub/releases/download/v2.2.0-rc1/hub_mac_386_v2.2.0-rc1.tar.gz"
```

Comment:
I really don't think we need to support 32 bit. I can't imagine someone on an ancient Mac using Boxen.